src/Selection.ts

```diff
@@ -20,6 +20,12 @@ import { Variables, buildVariableDefinitions } from "./Variables";
 
 type Element<T> = T extends Array<infer U> ? U : never;
 
+interface OperationOpts {
```

Comment:
Can we rename to `ToQueryOptions` please (i.e to make it explicit that it's an interface specifically for the `toQuery` method).